src/bdc/steps/enrich_custom_domains.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 Lucca Baumgärtner <lucca.baumgaertner@fau.de>
+
+import numpy as np
+import pandas as pd
+
+from bdc.steps.step import Step
+
+
+class EnrichCustomDomains(Step):
```

Comment:
Maybe this class should be called EnrichCustomDomainsStep to make clear it is a Pipeline step

Reply:
Tbh i feel like it's redundant, as this shows from both the class inheritance and the package structure.

Reply:
Fair enough